nav2_controller/include/nav2_controller/nav2_controller.hpp

```diff
@@ -227,8 +229,19 @@ class ControllerServer : public nav2_util::LifecycleNode
   double min_y_velocity_threshold_;
   double min_theta_velocity_threshold_;
 
+  // Speed limit parameters
+  std::string speed_limit_topic_;
```

Comment:
From looking at use, `speed_limit_topic_` doesn't need to be a class member, it can just be local to the configure function.

Reply:
Indeed, this is unnecessary member. Removed in https://github.com/ros-planning/navigation2/pull/2074/commits/34e694dbee7af9f805d2a7038b292466e44dc104#diff-db4cc04c6df9717814a17beefe3d9013c83753db07ef4a6bfc47e4ce97240d31L104

Reply:
This still appears in this PR's diff. Clicking on that hash gives me an error "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

Reply:
Should be removed in https://github.com/ros-planning/navigation2/pull/2074/commits/a18462c5fd0f3c608f18a6cdb204adbfb705b6bf#diff-ded6f35e50bcbab551153df0325f14c0160b309cd3175807c1763d2dad8da5f3L233